File: 00-Code/Android/NDK/BionicAPI/app/src/main/java/com/ztiany/bionic/MainActivity.java
package com.ztiany.bionic;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.ztiany.bionic.pthread.PthreadActivity;

public class MainActivity extends AppCompatActivity {

    private JniBridge mBridge;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mBridge = new JniBridge();
        setContentView(R.layout.activity_main);
    }

    public void testBionicApi(View view) {
        mBridge.testJni();
    }

    public void pthread(View view) {
        startActivity(new Intent(this, PthreadActivity.class));
    }
}

File: Android/00-Code/AAC-Sample/Navigation-Sample/src/main/java/com/ztiany/navigation/home/MainActivity.kt
package com.ztiany.navigation.home

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import androidx.navigation.Navigation
import com.ztiany.navigation.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun supportNavigateUpTo(upIntent: Intent) {
        Navigation.findNavController(this, R.id.home_nav_host_fragment).navigateUp()
    }

}
